test/controllers/api/tasks_controller_test.rb

```diff
@@ -30,20 +30,35 @@ class Api::TasksControllerTest < ActionController::TestCase
       describe 'GET /api/tasks/summary' do
         class DummyTestSummaryAction < Support::DummyDynflowAction
           # a dummy test action that do nothing
+          def plan(suspend = false)
+            plan_self :suspend => suspend
+          end
+
           def run
             # a dummy run method
+            if input[:suspend] && output[:suspended].nil?
+              output[:suspended] = true
+              suspend
+            end
           end
         end
 
         test_attributes :pid => 'bdcab413-a25d-4fe1-9db4-b50b5c31ebce'
         it 'get tasks summary' do
-          ForemanTasks.trigger(DummyTestSummaryAction)
+          triggered = ForemanTasks.trigger(DummyTestSummaryAction, true)
+          wait_for { ForemanTasks::Task.find_by(external_id: triggered.id).state == 'running' }
+          wait_for do
+            w = ForemanTasks.dynflow.world
+            w.persistence.load_step(triggered.id, 2, w).state == :suspended
```

Comment:
@adamruzicka I've added additional waiting just to ensure we're in suspended state, so that we make sure that the event gets to the right recipient later.